src/main/java/edu/umich/its/spe/SPEMaster.java

```diff
@@ -44,23 +45,27 @@
 public class SPEMaster {
 
 	static final Logger M_log = LoggerFactory.getLogger(SPEMaster.class);
+
+	// Keep maps of (some) specific sets of properties.
 	static HashMap<String,String> unirest;
+	static HashMap<String,String> emailMap;
+	static HashMap<String,String> repeatMap;
```

Comment:
I feel `repeatMap` can have a better name, it doesn't tell me anything more specific what its purpose for. A comment also would help

Reply:
Done.